gnosis/cowswap
# [presign] Remove cancelation of orders

# Summary

This PR removes the ability of cancelation of orders for pre-sign workflow.

Before:
![image](https://user-images.githubusercontent.com/2352112/132378115-363cf42f-2336-4dcf-bebd-4ab57e812173.png)


After:
![image](https://user-images.githubusercontent.com/2352112/132377896-8e75afa3-cf9a-45c1-8374-e298a852050b.png)


## not included
Onchain cancelation
#733 

# To Test

1. Create an order using pre-sigh (default, even for EOA in this PR)
2. Observe that now is not possible to cancel

# Background
SC wallets can't do offchain signing
